resident-default.properties

```diff
@@ -727,6 +734,7 @@ resident.template.summary.success.REVOKE_VID=gen-or-revoke-vid-success-summary
 resident.template.summary.success.GET_MY_ID=get-my-uin-card-success-summary
 resident.template.summary.success.VALIDATE_OTP=verify-my-phone-email-success-summary
 resident.template.summary.success.VID_CARD_DOWNLOAD=vid-card-download-positive-summary
+resident.template.summary.success.AUTHENTICATION_REQUEST=mosip.event.type.AUTHENTICATION_REQUEST
```

Comment:
why success, simmary and failure has the same value ?

Reply:
Changed success, summary and failure values